media-gfx/xpaint/xpaint-3.1.3.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs autotools xdg-utils
+
+DESCRIPTION="Image editor with tiff, jpeg and png support"
+HOMEPAGE="http://sf-xpaint.sourceforge.net/"
+SRC_URI="mirror://sourceforge/sf-xpaint/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="pgf tiff"
+# jpeg2k disabled for blocking media-libs/openjpeg:0 security cleanup, bug 735592
+
+RDEPEND="
+	media-libs/fontconfig
+	media-libs/freetype:2
+	media-libs/libjpeg-turbo:=
+	media-libs/libpng:0=
+	x11-libs/libICE
+	x11-libs/libX11
+	>=x11-libs/libXaw3dXft-1.6.2h[unicode]
+	x11-libs/libXext
+	x11-libs/libXft
+	x11-libs/libXmu
+	x11-libs/libXpm
+	x11-libs/libXt
+	sys-libs/zlib
+	pgf? ( media-libs/libpgf )
+	tiff? (
+		media-libs/jbigkit:0=
+		media-libs/tiff:0
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-libtool-clang.patch
+	"${FILESDIR}"/${P}-cflags-ldflags-strip.patch
+)
+
+src_prepare() {
+	default
+	sed -i -e 's/-O3 -s//g' util/Makefile || die
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable tiff) \
+		--disable-libdvipgm \
+		--disable-libopenjpeg
+}
+
+src_compile() {
+	# clean up
+	emake clean
+	emake -C util clean
+
+	# parallel make still fails sometimes
```

Comment:
We say that, but no -j1...?

Reply:
I think this this comment is about the next 2 lines, probably they were breaking parallel make so they are done before the main emake.
```
    emake substads
    emake xpaint.1
```

I compiled this many times using `-j12`, both with gcc and clang, and it seems fine.  I wouldn't add `-j1` unless someone reports a bug.

Reply:
Oh, okay, I see.